wpilibj/src/main/java/edu/wpi/first/wpilibj/AddressableLEDBuffer.java

```diff
@@ -0,0 +1,40 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj;
+
+/**
+ * Buffer storage for Addressable LEDs.
+ */
+public class AddressableLEDBuffer {
+  byte[] m_buffer;
```

Comment:
It seems like this should use a native-backed ByteBuffer instead?

Reply:
No, because a native backed buffer requires basically a jni call on every access. We need to do a lot of sets, and then only 1 read in jni.